### DISCUSSION TITLE
slash command options

### DISCUSSION THREAD

**Discussion by PingouinDuTurfu** [discordjs/discord.js#6743] [Q&A] [answered]
Hello, today I need help cause I don't find where is the problem... I have searched for hours but still not working please help me.
I want to put options in my SlashCommandBuilder how could I proceed? (I'm working on V13)

```
const { SlashCommandBuilder } = require('@discordjs/builders');

module.exports = {
	data: new SlashCommandBuilder()
		.setName('createroleassigner')
		.setDescription('Create a role selector.')
		.addStringOption(option => option.setName('user').setDescription('The user')),
	execute(interaction) {
		const optionsList = interaction.options.getString('user');
		console.log(optionsList); //alway null
	},
};
```

**Comment by jimfangx** [comment, marked as answer]
Everything looks fine. Make sure you are on Node.js 16+. Consider using the `.addUserOption` for inputting a user.

My tests: 
![image](https://user-images.githubusercontent.com/23324711/135743670-8fefaa66-f51a-40cb-8a8b-0642258296f1.png)

Post any errors you have.

**Reply by PingouinDuTurfu** [reply]
Thank you for helping me! You comfort me that the code was fine. And effectively that's working, I just didn't notice that when we enter a slash command line in Discord, we have to tab after the slash command and not simply put an argument.
